src/Sarif/Errors.cs

```diff
@@ -697,5 +698,27 @@ internal static void LogAnalysisTimedOut(IAnalysisContext context)
                     messageFormat: null,
                     configuredTimeout));
         }
+
+        public static void LogErrorPostingLogFile(IAnalysisContext context, string postUri)
+        {
+            if (context == null)
+            {
+                throw new ArgumentNullException(nameof(context));
+            }
+
+            context.RuntimeErrors |= RuntimeConditions.ExceptionPostingLogFile;
```

Comment:
Move above the nullcheck?

Reply:
can not, context.RuntimeErrors requires context not null